build-monitor-plugin/src/main/resources/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView/index.jelly

```diff
@@ -55,30 +55,14 @@
             <script src="${resourcesURL}/vendor/angular-slider-5.4.0/rzslider.min.js"></script>
 
             <j:if test="${it.collectAnonymousUsageStatistics()}">
-                <script>
-                    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-
-                    ga('create', 'UA-61694827-4', 'auto', {
-                        'userId':     '${it.installation.anonymousCorrelationId()}',
-                        'sampleRate': 1
-                    });
-
-                    ga('set', {
-                        'forceSSL':       true,
-                        'appName':        'Build Monitor',
-                        'appId':          'build-monitor-plugin',
-
-                        'appVersion':     '${it.installation.buildMonitorVersion()}',
-                        'appInstallerId': '${h.version}',
-
-                        'dimension1':     '${it.installation.size()}',
-                        'dimension2':     '${it.items.size()}',
-                        'dimension3':     '${it.installation.audience()}',
-                        'dimension4':     '${it.installation.anonymousCorrelationId()}'
-                    });
-
-                    ga('send', 'screenview', {screenName: 'Dashboard'});
-                </script>
+                <span class="build-monitor-ga-data-holder"
```

Comment:
I've noticed that adding a span here while we're inside the `head` tag makes this `span` move down to `body` along with the `script` tag produced by `st:adjunct` when the page is rendered. I think this can be fixed by moving the `span` down to `body` where it belongs, and then wrapping the content of `collect-usage-stats.js` into `window.addEventListener("DOMContentLoaded", ...)`

Reply:
https://github.com/jenkinsci/build-monitor-plugin/pull/1034/commits/f648f11eb046968b98c6cfa4948d8ec576bfd12f